doc/book/box/atomic/txn_mode_mvcc.rst

```diff
@@ -57,8 +57,8 @@ option to enable it via ``box.cfg``.
 
 ..  _txn_mode_mvcc-options:
 
-Transaction manager options
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Setting the transaction isolation level
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ..  note::
```

Comment:
The note mentions terms that haven't yet been introduced. Consider moving the note lower.

Reply:
Thanks, moved the note.